scripts/prepare-publish.cjs

```
@@ -0,0 +1,92 @@
+// Handle command line arguments
+const command = process.argv[2]
+
+switch (command) {
+  case 'clean':
+    cleanPackageForPublish()
+    break
+  case 'restore':
+    restorePackageAfterPublish()
+    break
+  default:
+    console.error('Usage: node scripts/prepare-publish.js [clean|restore]')
```

ESLint

[error] 'console' is not defined.

(no-undef)